jetstream/tests/data/test_events.ndjson

```diff
@@ -1,7 +1,7 @@
 {"submission_date": "2020-04-02", "client_id": "aaaa", "event_category": "normandy", "event_method": "enroll", "event_string_value": "test-experiment", "event_map_values": [{"key": "branch", "value": "branch1"}]}
 {"submission_date": "2020-04-03", "client_id": "bbbb", "event_category": "normandy", "event_method": "enroll", "event_string_value": "test-experiment", "event_map_values": [{"key": "branch", "value": "branch2"}]}
-{"submission_date": "2020-04-02", "client_id": "aaaa", "event_category": "normandy", "event_method": "exposure", "event_string_value": "test-experiment", "event_map_values": [{"key": "branch", "value": "branch1"}]}
-{"submission_date": "2020-04-03", "client_id": "bbbb", "event_category": "normandy", "event_method": "exposure", "event_string_value": "test-experiment", "event_map_values": [{"key": "branch", "value": "branch2"}]}
+{"submission_date": "2020-04-02", "client_id": "aaaa", "event_category": "normandy", "event_method": "exposure", "event_string_value": "test-experiment", "event_map_values": [{"key": "branchSlug", "value": "branch1"}]}
+{"submission_date": "2020-04-03", "client_id": "bbbb", "event_category": "normandy", "event_method": "exposure", "event_string_value": "test-experiment", "event_map_values": [{"key": "branchSlug", "value": "branch2"}]}
```

Comment:
I noticed in @danielkberry's queries that `expose` is (sometimes?) used for `event_method`, so may be worth changing one of these test events to `expose` to cover both options.

```suggestion
{"submission_date": "2020-04-03", "client_id": "bbbb", "event_category": "normandy", "event_method": "expose", "event_string_value": "test-experiment", "event_map_values": [{"key": "branchSlug", "value": "branch2"}]}
```